### PR TITLE
Cleanup Addon

### PR DESCRIPTION
**Pull request by AlwinEsch**
bildschirmfoto von 2018-12-01 17-13-00

Also is a Fanart and a icon added, not the best, but better as nothing ;)

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -1,28 +1,10 @@
 Format: http://dep.debian.net/deps/dep5
-Upstream-Name: screensaver.asteroids
+Upstream-Name: screensaver.stars
```
Comment (Rechi) [inline debian/copyright]:
```suggestion
Upstream-Name: screensaver.asteroids
```

File: debian/copyright [hunk 2]
```diff
@@ -1,28 +1,10 @@
...
+Source: https://github.com/xbmc/screensaver.stars
```
Comment (Rechi) [inline debian/copyright]:
```suggestion
Source: https://github.com/xbmc/screensaver.asteroids
```

File: src/shaders/Matrix.cpp [hunk 3]
```diff
@@ -21,18 +21,18 @@
 #include "Matrix.h"
 #include <cmath>
 
-#define MODE_WITHIN_RANGE(m) ((m >= 0) && (m < (int)MM_MATRIXSIZE))
+#define MODE_WITHIN_RANGE(m)       ((m >= 0) && (m < (int)MM_MATRIXSIZE))
```
Comment (Rechi) [inline src/shaders/Matrix.cpp]: The modifications in this file aren't an improvement, those should be removed.

**Comment by AlwinEsch** [comment]
Works on Linux, Windows and Android.

**Pull request merged**